src/main/java/com/google/devtools/build/lib/actions/ExecutionRequirements.java

```diff
@@ -154,6 +154,30 @@ public String parseIfMatches(String tag) throws ValidationException {
             return null;
           });
 
+  /** How many extra resources an action requires for execution. */
+  public static final ParseableRequirement RESOURCES =
+      ParseableRequirement.create(
+          "resources:<str>:<float>",
+          Pattern.compile("resources:(.+:.+)"),
```

Comment:
I'd prefer not to do this:
This would require modification to the `ParseableRequirement create` method above, which assumes a single group is matched. My inclination is to keep this PR as small as possible.

Reply:
@larsrc-google is this acceptable? Was your suggestion phrased this way to be polite? Is it resolved by my expressed preference here?

Reply:
You're right, changing that would be overkill.